test/images/scripts/rhel-7.setup

```diff
@@ -135,7 +124,23 @@ debuginfo-install -y storaged storaged-lvm2 storaged-iscsi
 
 # only install mock and build if TEST_SOURCE is set
 if [ -n "$TEST_SOURCE" ]; then
-    yum -y install mock rpm-build
+    yum -y install rpm-build
+
+    # enable epel for mock
+    if [ ! -f "$SKIP_REPO_FLAG" ]; then
+        mkdir /tmp/dep
+        cd /tmp/dep
+        yum -y install wget
+        wget -T 15 -t 4 http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Comment:
We can remove wget afterwards. I'd rather not hack something with curl just to get wget behavior - the fedoraproject server can be a bit flaky.